tests/Functional/Fixtures/Symfony/AppCache.php

```diff
@@ -25,4 +29,14 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
 
         return $response;
     }
+
+    /**
+     * Made public to allow event subscribers to do refresh operations.
+     *
+     * {@inheritDoc}
+     */
+    public function fetch(Request $request, $catch = false)
```

Comment:
@stof this is the solution i came up with, because of the problem i explain on stackoverflow. its annoying but i guess its really a bug (or illogic decision) of traits vs interfaces.